Tests/CMakeLists.txt

```diff
@@ -15,12 +16,8 @@ APPEND_SOURCE_FILES(TEST_SOURCES MeshLib)
 APPEND_SOURCE_FILES(TEST_SOURCES MeshGeoToolsLib)
 APPEND_SOURCE_FILES(TEST_SOURCES NumLib)
 
-set(TEST_SOURCES ${TEST_SOURCES}
-	FileIO/TestGLIReader.cpp
-	FileIO/TestCsvReader.cpp
-)
 if(QT4_FOUND)
-	set(TEST_SOURCES ${TEST_SOURCES} FileIO/TestXmlGmlReader.cpp)
+	set(TEST_SOURCES ${TEST_SOURCES} FileIO_Qt/TestQtGmlInterface.cpp)
```

Comment:
Just use `APPEND_SOURCE_FILES` here as well (for consistency and the case there will be more test files in the future).